rom/laghos_rom.hpp

```diff
@@ -56,6 +56,15 @@ struct ROM_Options
 
     std::string *basename = NULL;
 
+    std::string basisIdentifier = "";
+    double greedyTol = 1.0;
```

Comment:
The greedy parameters could use some documentation. There is some in the options parsing in laghos.cpp but some more detailed descriptions could be added here.